unpack-build-maven-plugin/src/main/java/org/kie/unpackbuildplugin/OverrideDependenciesMojo.java

```diff
@@ -0,0 +1,141 @@
+package org.kie.unpackbuildplugin;
+
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.model.Dependency;
+import org.apache.maven.model.DependencyManagement;
+import org.apache.maven.model.Model;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.StringUtils;
+import org.kie.unpackbuildplugin.util.PomManipulationUtils;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@Mojo(name = "override-dependencies", defaultPhase = LifecyclePhase.GENERATE_SOURCES)
+public class OverrideDependenciesMojo extends AbstractMojo {
```

Comment:
Please add a JavaDoc (it may be something short), describing what the Mojo does.

Reply:
Done.